content/200-orm/200-prisma-client/500-deployment/301-edge/450-deploy-to-cloudflare.mdx

```diff
@@ -536,6 +545,193 @@ npx wrangler deploy
 
 The command will output the URL where you can access the deployed Worker.
 
-### D1
+### Cloudflare D1
+
+If you are using a D1 database, you need to:
+
+- use the `@prisma/adapter-d1` database adapter (via the `driverAdapters` Preview feature)
+- set `sqlite` as the `datasource` provider in your Prisma schema
+- manually generate SQL statements for schema changes using `prisma migrate diff` but execute them using [D1's migration system](https://developers.cloudflare.com/d1/reference/migrations/)
```

Comment:
I'd add a recommendation to use the latest version of Wrangler (e.g., `wrangler@^3.39.0`), otherwise the `--remote` flag in `wrangler d1 migrations apply <DATABASE_NAME> --remote` won't be available.